### PR TITLE
ui: add dark mode

### PR DESCRIPTION
**Pull request by BugenZhao**

### CODE REVIEW

**Comment by skyzh** [comment]
image

Please add a background to our logo.

File: src/App.tsx [hunk 1]
```diff
@@ -15,84 +15,83 @@ import {
 import ThreadList from "./components/lists/ThreadList"
 import Login from "./components/views/Login"
 import Settings from "./components/views/Settings"
-import { theme } from "./theme"
 
 function App() {
+  const navBgColor = useColorModeValue("gray.50", "gray.900")
+  console.log({ navBgColor })
```
Comment (skyzh) [inline src/App.tsx:21]: Remove logging.

**Review by skyzh — COMMENTED** [review]
What about adding an option in `settings` for dark mode?

**Comment by hebingchang** [comment]
Please set `backgound-color` for the navigation bar.

image

File: src/theme.ts [hunk 2]
```diff
@@ -5,4 +5,8 @@ export const theme = extendTheme({
     semibold: 500,
     bold: 500,
   },
+  config: {
+    initialColorMode: "light",
+    useSystemColorMode: false,
```
Comment (hebingchang) [inline src/theme.ts:10]: Why not using system color mode?

Comment (BugenZhao) [reply src/theme.ts:10]: We will consider giving the option to users in the future :)

**Pull request merged**